Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -D linkcheck_anchors=
```

Comment:
Please add a comment that indicates why we do this (maybe with a link to this PR).